flexget/plugins/clients/rtorrent.py

```diff
@@ -387,6 +425,29 @@ def _build_options(self, config, entry, entry_first=True):
                 elif entry_value:
                     options[opt_key] = entry.render(entry_value)
 
+        # Add custom fields on to options
+        entry_custom_fields = entry.get('custom_fields')
+        config_custom_fields = config.get('custom_fields')
+
+        if entry_custom_fields is None:
+            entry_custom_fields = {}
+
+        if config_custom_fields is None:
+            config_custom_fields = {}
+
+        merged_custom_fields = {}
+        if entry_first:
+            merged_custom_fields.update(entry_custom_fields)
+            merged_custom_fields.update(config_custom_fields)
+        else:
+            merged_custom_fields.update(config_custom_fields)
+            merged_custom_fields.update(entry_custom_fields)
```

Comment:
It looks to me like `entry_first` is meant to mean that entry fields take priority, so I think the merge order here is reversed (whatever you call update with last will overwrite the previous values.) You can also simplify the syntax a bit:
```suggestion
        if entry_first:
            merged_custom_fields = {**config_custom_fields, **entry_custom_fields}
        else:
            merged_custom_fields = {**entry_custom_fields, **config_custom_fields}
```